Apps/Playground/0.65/npm_run_start_loop.cmd

```diff
@@ -0,0 +1,6 @@
+:Loop
```

Comment:
What is this? If we want some tooling like this, we should put it in a shared node script somewhere.

Reply:
Oops, that's not supposed to be in the PR. :)